roles/validate-app/tasks/main.yaml

```diff
@@ -8,32 +8,29 @@
       command: "{{ openshift.common.client_binary }} new-project validate"
 
     - name: Create Hello world app
-      shell: "{{ openshift.common.client_binary }} new-app https://github.com/openshift/cakephp-ex.git"
+      shell: "{{ openshift.common.client_binary }} new-app cakephp-mysql-example"
```

Comment:
I'd say to set cakephp-mysql-example to a variable to avoid rewritting it a few times (and change it if needed)